### ISSUE TITLE
Questions regarding specific ligands in the PLB

### ISSUE THREAD

**Issue by hannahbaumann** [openforcefield/protein-ligand-benchmark#101]
I have some questions that came up when going through the PLB:

1. in some datasets, some of the ligands are deposited more than ones, but they have the same name in the .sdf . It would be helpful to give them different names (will list details of those cases below)

2. Target: tnks2
Ligand lig_7 is negatively charged, the amide N does not have a hydrogen. All other ligands in the series have the same core and do have the hydrogen there.
 
<img width="449" alt="Screen Shot 2023-11-02 at 3 58 49 PM" src="https://github.com/openforcefield/protein-ligand-benchmark/assets/43765638/8b3a209b-a69a-4c98-9ada-7b2ce91cfbe2">

 3. Target: SYK2
 Ligands lig_CHEMBL3265030_n and lig_CHEMBL3265035 have both two copies of the ligand in the .sdf, those are two "stereoisomers" (around a charged nitrogen, so not a real stereo center, but an "constant pH MD stereo center"=). At what pH were these structures prepared? Since the nitrogen is next to the phenyl ring, could it be neutral around neutral pH? Both in the PLB from Hahn et al as well as in the latest Schrodinger benchmark (https://github.com/schrodinger/public_binding_free_energy_benchmark/blob/a65d86b782a678c75388566ce00446e45283ee63/fep_benchmark_inputs/structure_inputs/merck/syk_4puz_fullmap_ligands.sdf) these two ligands are neutral (and therefore also don't have a "stereo center" at that nitrogen.
 
<img width="430" alt="Screen Shot 2023-11-02 at 4 06 31 PM" src="https://github.com/openforcefield/protein-ligand-benchmark/assets/43765638/e8dec0bd-e821-473d-a0ff-81a4668b1e2b">
<img width="408" alt="Screen Shot 2023-11-02 at 4 06 39 PM" src="https://github.com/openforcefield/protein-ligand-benchmark/assets/43765638/1974ea27-e3be-4d7c-8dcb-a38c49feaa90">

 
 
 4. Target eg5:
 ligand lig_CHEMBL1084935 has a charged tertiary amine and therefore also a "stereocenter". Both "stereo isomers" are deposited . Both have the same name which makes it difficult to separate them from each other in e.g. a ligand network. Here the question I have is whether to always include both in the calculations, or whether it makes sense to run a transformation between them, figure out which is the more favorable one and only include that one in future benchmark studies.
 
<img width="359" alt="Screen Shot 2023-11-02 at 4 23 24 PM" src="https://github.com/openforcefield/protein-ligand-benchmark/assets/43765638/f8ee39d9-c184-4440-8b41-868fcf93a6e4">

 
<img width="346" alt="Screen Shot 2023-11-02 at 4 23 29 PM" src="https://github.com/openforcefield/protein-ligand-benchmark/assets/43765638/531d72a5-8a51-4d63-9a54-2302d48f7015">

 
 
 5. Target P38:
 Ligand lig_p38a_2ff is deposited with two orientations of the cyclohexyl ring. Would it make sense to see whether one of them is more favorable and only include that one or to see if the conformations interconvert in MD?
 
<img width="493" alt="Screen Shot 2023-11-02 at 4 25 28 PM" src="https://github.com/openforcefield/protein-ligand-benchmark/assets/43765638/2e103704-6903-4d4f-a5b6-10ed022e5f90">
<img width="423" alt="Screen Shot 2023-11-02 at 4 25 34 PM" src="https://github.com/openforcefield/protein-ligand-benchmark/assets/43765638/6ad6750a-f720-4e20-948e-c9e15fea34c0">

 
 
 6. Target: Thrombin
 Ligand lig_7d is deposited 3x in the PLB, in three different stereo isomers. Do we know whether the experimental measurement was run with a single stereo isomer or a mixture? If this is unknown, maybe it would make sense to remove this ligand from the PLB?
 
 7. Target: cmet
 Ligand lig_CHEMBL3402761_1_21 has a tertiary amine which is not positively charged (but used to be protonated in the Hahn et al. PLB). Is the pKa maybe near the assay pH? Would one have to account for both protonation states in the benchmark calculations or would it make sense to just stick with one protonation state and use that for all benchmarking exercises?
 
<img width="460" alt="Screen Shot 2023-11-02 at 4 27 44 PM" src="https://github.com/openforcefield/protein-ligand-benchmark/assets/43765638/9bd523e2-5e00-4371-96f6-20f90544bd3b">

 8. Target thrombin
 All ligands have a primary amine which is not charged, while it was protonated in the Hahn et al. PLB. Similar case/question as above. 
 
<img width="302" alt="Screen Shot 2023-11-02 at 4 29 01 PM" src="https://github.com/openforcefield/protein-ligand-benchmark/assets/43765638/a6ccd5b9-66a9-4ae2-ab87-cd682a88bd46">

Looking forward on your input on these questions!

**Comment by hannahbaumann** [comment]
@ijpulidos : Thank you for looking into this!
Since we needed to make a (short term) decision for openfe, I worked on some of the changes mentioned about in this PR (https://github.com/OpenFreeEnergy/protein-ligand-benchmark/pull/10), off of a fork of this repo. The closed issues give a brief description of what I did. I'm happy to work together to bring the changes that you mentioned above into this repo (either merging back the parts of the fork PR that you want to include or by creating a branch/PR directly here). For changing the protonation states I had mostly use openeye tools. Would you prefer to instead continue using the tools that were used to prep all the files?
Please let me know what would be the best way to proceed here!

**Comment by ijpulidos** [comment]
@hannahbaumann Yes, that sounds great. Happy to including the improvements in this dataset. I think we probably want to shift to use the OpenEye tools, since they seem to be working fine for our objectives. Also related to https://github.com/openforcefield/protein-ligand-benchmark/issues/104#issuecomment-1830332293

**Comment by hannahbaumann** [comment]
@ijpulidos : I had completely forgotten to reply here, sorry about that! I'm still happy to open up a PR regarding the changes here. Do you think that still makes sense or will you switch to the OpenEye preparation soon and add the new structures here? 

In the meantime one of our collaborators also ran a pka prediction for these molecules using the Simulations Plus ADMETpredictor V.11 (in the context of the OpenFF consortium for method development). Please let me know if you would also be interested in those results (you can also find some of those in these issues):

https://github.com/OpenFreeEnergy/protein-ligand-benchmark/issues/2#issuecomment-1846874182
https://github.com/OpenFreeEnergy/protein-ligand-benchmark/issues/1#issuecomment-1846871869
https://github.com/OpenFreeEnergy/protein-ligand-benchmark/issues/6#issuecomment-1846868433
https://github.com/OpenFreeEnergy/protein-ligand-benchmark/issues/7#issuecomment-1846878089
